token/delegation/ipld.go

```diff
@@ -193,8 +194,16 @@ func FromIPLD(node datamodel.Node) (*Token, error) {
 }
 
 func (t *Token) toIPLD(privKey crypto.PrivKey) (datamodel.Node, error) {
-	var sub *string
+	// sanity check that privKey and issuer are matching
```

Comment:
if it stays here (aka, not directly in the envelope package), it needs to be replicated in invocation